api/proto/teleport/usertasks/v1/user_tasks.proto

```diff
@@ -85,4 +85,8 @@ message DiscoverEC2Instance {
   string discovery_group = 7;
   // SyncTime is the timestamp when the error was produced.
   google.protobuf.Timestamp sync_time = 8;
+
+  // AccountID and Region were moved into the DiscoverEC2 message.
+  reserved 3, 4;
```

Comment:
move this to the top of the message as it's easier to read/follow

Reply:
I prefer them at the bottom because that's where you usually add new fields, so the forbidden identifiers are right below your cursor.

But that's fine, we have linters anyway 👍